docs/src/development/UOS.md

```diff
@@ -275,7 +275,7 @@ Thus, the transaction structure could also be represented as:
 
 Bold-marked transaction pieces are used in the blob for which the signature is
 produced. If the blob is short, 257 bytes or below, the signature is produced
-for it as is. For blobs longer than 257 bytes, 32 byte hash (`blake2-rfc`) is
+for it as is. For blobs longer than 257 bytes, 32 byte hash (`blake2_256`) is
```

Comment:
Do you mean changing direct use of `blake2-rfc` crate to Substrate wrapper? Or this is just for more clear wording here? The `transaction_signing` (where the long transaction hashing is done) still has direct `blake2-rfc`.

Reply:
checked signing westend lorem ipsum remark on-chain by Alice//westend using this branch; works correctly.